python/ingestion/merge_utils.py

```diff
@@ -126,25 +128,37 @@ def merge_state_ids(df, keep_postal=False):
     return df
 
 
-def merge_pop_numbers(df, demo: str, loc: str):
+def merge_pop_numbers(df, demo: Literal['age', 'race', 'sex'], loc: Literal['county', 'state', 'national']):
```

Comment:
Could we use the `SEX_RACE_AGE_TYPE` and `GEO_TYPE` types for this function and others also using it?

Reply:
So what I found was, when I used a variable as a type, the vscode intellisense didn't render the underlying types (like it does for typescript). So on hover you only saw the SEX_AGE_RACE_TYPE instead of the actual strings needed.  I definitely think it's helpful to be able to hover a function call and see the specific strings required to make that fn work, otherwise we'd have to click into the function and then hover that inner type variable. There may be a way to fix this in vs code; because definietly I'd rather use reusable types

Defining function signature explicitly:
<img width="514" alt="Screen Shot 2023-03-20 at 2 09 26 PM" src="https://user-images.githubusercontent.com/41567007/226454868-c54a9422-15fc-4187-a8e1-c93cad18e4c3.png">

Using variable in function signature (for demographic breakdown): 
<img width="514" alt="Screen Shot 2023-03-20 at 2 12 42 PM" src="https://user-images.githubusercontent.com/41567007/226454971-72b3d3c3-a32e-4122-8a2a-ccdbe32c0cc5.png">